experiments/react-native-uifabric/src/components/PersonaCoin/PersonaCoin.helpers.ts

```diff
@@ -0,0 +1,97 @@
+  dnd: { uri: 'data:image/png;base64,iVBORw0KGgoAAAANSUhEUgAAABgAAAAYCAMAAADXqc3KAAAAQlBMVEUAAADpABvpABvpABvpABvpABvpABvpABvpABvpABvpABvpABvpABvpABvpABvpABvpABvpABv////wfYT86errJjiR9OLxAAAAEXRSTlMA4/CzeQj602seEhEk89jKakfnNfsAAACKSURBVCjPhZFbDoQgDEWBQkHBR2d0/1udEppR0cbzeU9SSq95I69gcZphzZc4eEeC8+HIh0gn4vDPLV2wYsJCHUub5umGr/no7sKNLBI9kFjAkwAWbaX9K+xtMRZIle0jbFRBFpMmZmWU8ri+rv5B/SQmxD6PUknpz16MUJSi9GqZnMAiWkjZvPADxcoYalVfnOMAAAAASUVORK5CYII='},
+  blocked: {uri: 'data:image/png;base64,iVBORw0KGgoAAAANSUhEUgAAABgAAAAYCAMAAADXqc3KAAAAe1BMVEUAAADpABvpABvpABvpABvpABvpABvpABvpABvpABvpABvpABvpABvpABvpABvpABvpABv////pABv83uHqCiTsJz7rGjP7zNHvRVj/+vv+8vP6x83sIzv2mqXsKD/7zNL5v8b3oqz2maTuP1P82Nz819vtKUDrGjLrGTK2dyGvAAAAEXRSTlMA8dSzeQj64mseEhHkJMpq5T3coEAAAADSSURBVCjPhdHZFoIgFAVQGQRLGzhQiaLNw/9/YZRk4IvncZ+17uJys7mUGyYFoWxTJpxzghDC878vKFBftLX6UgN0MXqF/qRCTg5VaPI1ro36Re+MWQ/TOPrEXw7840uCNvb9oQVZ+qJAnbpSNQpfMHQTVx2YLyT0xJWG9IWAnbiyEL4gaCKPCopb7P9RDOeJqzNYeG7i3+eGBVMPC2Yczpn9cfTmAT58IoV5Rn4FDSdZbY1rxzk9qpXHofkcqtO2uXfhUHOn9SkLJoWQrCizmbwBa5IbuX8ugpIAAAAASUVORK5CYII='},
+  busy: {uri: 'data:image/png;base64,iVBORw0KGgoAAAANSUhEUgAAABgAAAAYCAMAAADXqc3KAAAANlBMVEUAAADpABvpABvpABvpABvpABvpABvpABvpABvpABvpABvpABvpABvpABvpABvpABvpABvpABvsgIdoAAAAEXRSTlMA8eLUEQj6s2sheXrlyrJqHBqb3nMAAAB/SURBVCjPhZFRDsQgCEQFEdS223L/yy7JummtGN/nvASUCSvoFGSIclIXpwragC3deYn6IJZ/fqF24NXmHPri+E2rOlCDQTAKIBNZHbKJjyfEBHoCTbAn2ATMRPRHzZfPn+t/cH4SIw3rY2p17H2+l0VRRtr8ag3KgswomcKCL1NmGBp9lqsZAAAAAElFTkSuQmCC'}
+};
+
+export type PersonaSizeConfig = {
+  physicalCoinSize: number;
+  iconSize: number;
+  initialsFontSize: number;
+};
+
+export function getSizeConfig(size: PersonaSize): PersonaSizeConfig {
+  switch (size) {
+    case PersonaSize.size8:
+      return { physicalCoinSize: 8, iconSize: 0, initialsFontSize: 4 };
+    case PersonaSize.size24:
+      return { physicalCoinSize: 24, iconSize: 8, initialsFontSize: 10 };
+    case PersonaSize.size32:
+      return { physicalCoinSize: 32, iconSize: 8, initialsFontSize: 12 };
+    case PersonaSize.size40:
+      return { physicalCoinSize: 40, iconSize: 12, initialsFontSize: 14 };
+    case PersonaSize.size48:
+      return { physicalCoinSize: 48, iconSize: 12, initialsFontSize: 16 };
+    case PersonaSize.size56:
+      return { physicalCoinSize: 56, iconSize: 16, initialsFontSize: 18 };
+    case PersonaSize.size72:
+      return { physicalCoinSize: 72, iconSize: 20, initialsFontSize: 20 };
+    case PersonaSize.size100:
+      return { physicalCoinSize: 100, iconSize: 28, initialsFontSize: 36 };
+    case PersonaSize.size120:
+      return { physicalCoinSize: 120, iconSize: 32, initialsFontSize: 40 };
+  }
+}
+
+export function getPresenceIconSource(presence: PersonaPresence): ImageURISource {
+  return presenceIconCache[PersonaPresence[presence]];
+}
+
+function personaCoinColorToHexCode(personaCoinColor: PersonaCoinColor): string {
+  switch (personaCoinColor) {
+    case PersonaCoinColor.lightBlue:

```

Comment:
Same comment as above, use discrete unions of strings and direct indexing into an object rather than a case statement.

Reply:
Also should this table be in the tokens?  Such that a user can override one of the colors if they want?

Reply:
I don't think we want to allow that. 'lightBlue' should show as light blue, instead of a different color. If they want to change color, they can provide the `coinColor` token.